example/src/app/a-bit-more-complex-component/__snapshots__/a-bit-more-complex.component.spec.ts.snap

```diff
@@ -0,0 +1,52 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ABitMoreComplexComponent snapshot test 1`] = `
+<app-a-bit-more-complex
+  anotherVar="false"
+  someVar={[Function Boolean]}
+>
+  
+      <div
+    class="kek"
+  >
+    
```

Comment:
I don't like this formatting. Do you mind adding some format tool like prettier or stuff?

Reply:
Hi, I found what jest use to format their snapshot. It is called pretty-format. Perhaps you can take a look https://www.npmjs.com/package/pretty-format

Reply:
They seem to use [pretty-format](https://github.com/facebook/jest/tree/master/packages/pretty-format#usage-with-plugins) with a custom React plugin

Reply:
Hi @sharikovvladislav , @barretodavid,
There is a solution for this. Can you guys please check #86 discussion ?